lms/static/scripts/frontend_apps/components/test/BookList-test.js

```diff
@@ -0,0 +1,68 @@
+import { mount } from 'enzyme';
+import { createElement } from 'preact';
+
+import BookList from '../BookList';
+
+describe('BookList', () => {
+  const bookData = [
+    {
+      id: 'test-book',
+      title: 'Test book',
+      cover_image: 'https://example.com/test.jpg',
+    },
+  ];
```

Comment:
One day we might want to talk about fixtures for our apps...it's something I've had back-of-mind for a while.